### ISSUE TITLE
[BugFix] [RHEL/6] [/usr/share/scap-security-guide/remediation_functions] package_command install / remove shows "line 381: /usr/bin/rpm: No such file or directory" warning when run on RHEL-6. Fix this by providing more universal path to "rpm" executable

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#972]
Current output when running latest ```master``` on RHEL-6 system:
```
# rpm -q scap-security-guide
scap-security-guide-0.1.28-1.el6.noarch
# pwd
/root/scap-security-guide/RHEL/6/input/remediations/bash
# bash --norc smartcard_auth.sh 
/usr/share/scap-security-guide/remediation_functions: line 381: /usr/bin/rpm: No such file or directory
Loaded plugins: product-id, rhnplugin, search-disabled-repos, subscription-manager
This system is receiving updates from RHN Classic or RHN Satellite.
Setting up Install Process
Package esc-1.1.0-26.el6.x86_64 already installed and latest version
Nothing to do
/usr/share/scap-security-guide/remediation_functions: line 381: /usr/bin/rpm: No such file or directory
Loaded plugins: product-id, rhnplugin, search-disabled-repos, subscription-manager
This system is receiving updates from RHN Classic or RHN Satellite.
Setting up Install Process
Package pam_pkcs11-0.6.2-14.el6.x86_64 already installed and latest version
Nothing to do
Starting PC/SC smart card daemon (pcscd): 
grep: : No such file or directory
```

Current result:

--
```
/usr/share/scap-security-guide/remediation_functions: line 381: /usr/bin/rpm: No such file or directory
```
warning is shown when ``package_command install / remove``` SSG remediation command is used on RHEL-6 system.

Expected result:

--
No warning. ```rpm``` executable is located properly by ```package_command install / remove``` SSG remediation command.

Additional info:

--
The reason for this warning being ```rpm``` on RHEL-6 is installed under ```/bin/rpm``` path (instead of ```/usr/bin/rpm``` like it's the case for RHEL-7 and Fedora systems). But the ```/bin/rpm``` path is still present / available on RHEL-7 and Fedora systems:
```
$ file /bin/rpm
/bin/rpm: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=ae631a23edf449891442b04560b05647c312f56e, stripped
```
but it's just symlink to ```/usr/bin/rpm```. So IMHO we should prefer ```/bin/rpm``` path in ```package_command``` SSG remediation tool, to remove this warning / the tool to work everywhere.

**Comment by iankko** [comment]
@redhatrises 
Could you hopefully have a look @ fixing this? Thanks, Jan

**Comment by redhatrises** [comment]
@iankko this was fixed in by @isimluk in #975
